code_schemes/idp_camp.json

```diff
@@ -0,0 +1,102 @@
+{
+  "SchemeID": "Scheme-0fe57c6d",
+  "Name": "IDP Camp",
```

Comment:
Consider 'In IDP Camp' to separate for a potential scheme enumerating IDP camps.

Reply:
renamed